ui.frontend/webpack.npm.js

```diff
@@ -47,7 +47,7 @@ module.exports = merge(
             {
               loader: 'babel-loader',
               options: {
-                configFile: './babel.config.npm.js',
+                presets: ['@babel/typescript']
               },
```

Comment:
**Thought**

I saw that some imports were not used, could you also delete them? Thanks!

```
const { merge } = require('webpack-merge')
const webpack = require('webpack') <---------
const common = require('./webpack.common.js') <---------
const path = require('path')
const TSConfigPathsPlugin = require('tsconfig-paths-webpack-plugin')
const MiniCssExtractPlugin = require('mini-css-extract-plugin') <---------
const { CleanWebpackPlugin } = require('clean-webpack-plugin')
const CopyWebpackPlugin = require('copy-webpack-plugin') <---------
```